securedrop/journalist_templates/admin_add_user.html

```diff
@@ -6,17 +6,39 @@
 
 <form method="post">
   <input name="csrf_token" type="hidden" value="{{ csrf_token() }}">
-  <input name="password" type="hidden" value="{{ password }}">
-  <p><input name="username" type="text" placeholder="{{ gettext('Username') }}"></p>
+  <p>
+    {{ form.password(value=password, id=False) }}
```

Comment:
I'm curious about when an error can happen with the password field ? It has been generated and unless the user deliberately modifies the value (debugger etc.), it looks like it is going to always be good.

Reply:
Ah, true - removed